bin/cli.js

```diff
@@ -1,9 +1,15 @@
 #!/usr/bin/env node
 'use strict';
 
+
+const extensionArgs = ['--extensions', 'hbs'];
 const transformName = process.argv[2];
 const args = process.argv.slice(3);
 
-console.log(transformName, args);
+console.log(args);
 
-require('codemod-cli').runTransform(__dirname, transformName, args);
+require('codemod-cli').runTransform(
+    __dirname, 
+    transformName, 
+    extensionArgs.concat(args)
```

Comment:
This needs to be tweaked still:

```suggestion
    args,
    'hbs'
```